.github/workflows/trigger_release.yml

```
@@ -56,6 +56,9 @@ jobs:
       - name: tune linux network
         run: sudo ethtool -K eth0 tx off rx off
 
+      - name: update pnpm
+        run: pnpm i -g pnpm
```

```suggestion
      - run: corepack enable
```

Add this here and remove from line 49